### PR TITLE
Update guest kernel to 5.12

### PR DESCRIPTION
**Pull request by rbradford**
Update recommended guest kernel to 5.12 in integration tests and documentation.

### CODE REVIEW

File: docs/fs.md [hunk 1]
```diff
@@ -35,13 +35,10 @@ The `cache=none` option should be the default when using `virtiofsd` with the __
...
 
-### The kernel
+### Kernel support
 
-In order to leverage __virtio-fs__ support from within the guest, and because the code has not been merged in upstream Linux kernel yet, it is required to build a custom kernel embedding the patches.
-
-The following branch `virtio-fs-virtio-iommu` on the repository https://github.com/cloud-hypervisor/linux.git includes all the needed patches to support __virtio-fs__.
-
-Make sure to build a kernel out of this branch that can be then used to boot the VM.
+Modern Linux kernels starting (at least v5.10) have support for virtio-fs. Use
```
Comment (sboeuf) [inline docs/fs.md:40]: 5.9

**Review by sboeuf — APPROVED** [review]
LGTM except 1 nit about the kernel version

File: docs/fs.md (hunk 1 above)
Comment (rbradford) [reply docs/fs.md:40]: We never tested with 5.9 so that's why I chose to say 5.10

Comment (sboeuf) [reply docs/fs.md:40]: Sounds good!

**Pull request merged**